eclipse-theia/theia
# wip: expose 'top-menu' bug

#### What it does


The following **wip** pull-request exposes an issue previously reported on our [forum](https://community.theia-ide.org/t/invisible-toolbar-menus/1923/11). When starting the application, we end up in a state when the top-level menu (main menu) is not visible:

![image](https://user-images.githubusercontent.com/40359487/131679380-46d23730-ddd9-4f67-a8da-0121e179d5f4.png)


#### How to test


1. clean environment (`rm -rf ~/.theia`)
2. build the `example-browser` application
3. start the application, and notice the missing top-level menu

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
